src/controllers/sessionController.ts

```diff
@@ -46,6 +51,20 @@ export default class SessionController {
     | IResponseSuccessJson<PublicSession>
   > =>
     withUserFromRequest(req, async user => {
+      const zendeskSuffix = await pipe(
+        profileWithValidNameAndEmailAddressOrError(this.profileService, user),
+        TE.bimap(
+          _ => crypto.randomBytes(4).toString("hex"),
```

Comment:
keep the same number of bytes as for validated email
```suggestion
          _ => crypto.randomBytes(8).toString("hex"),
```

Reply:
We shouldn't, 8 random bytes would be converted to a 16 chars hex string :)

Reply:
Sorry, my bad